packages/phone-number-privacy/src/config.ts

```diff
@@ -1,9 +1,11 @@
+import BigNumber from 'bignumber.js'
 import * as functions from 'firebase-functions'
 import logger from './common/logger'
 
 export const DEV_MODE =
   process.env.NODE_ENV !== 'production' || process.env.FUNCTIONS_EMULATOR === 'true'
 
+const CUSD_VALUE = 1000000000000000000
```

Comment:
Rather than hardcoding this, consider using the `toWei` method in web3

Reply:
cool thanks